src/SFA.DAS.ApprenticeFeedback.Application.UnitTests/Queries/WhenRequestingFeedbackTargetsForUpdate.cs

```diff
@@ -0,0 +1,437 @@
+                                    .ToList();
+
+                    apprenticeFeedbackTargets.Should().BeEmpty();
+                }
+
+                [Test]
+                [AutoMoqData]
+                public void When_ApprenticeshipStarted_Then_ReturnApprenticeship(
+                    [Frozen(Matching.ImplementedInterfaces)] ApprenticeFeedbackDataContext context,
+                    UtcTimeProvider dateTimeHelper
+                    )
+                {
+                    context.Add(AFT_Completed("Provider Name"));
+                    context.SaveChanges();
+
+                    var apprenticeFeedbackTargets = ((IApprenticeFeedbackTargetContext)context)
+                                    .Entities
+                                        .HasStarted(dateTimeHelper)
+                                    .ToList();
+
+                    apprenticeFeedbackTargets.Should().HaveCount(1);
+                }
+            }
+
+            public class StatusNotCompletedOrPermanentlyWithdrawn
+            {            
+                [Test]
+                [AutoMoqData]
+                public void When_ApprenticeshipStatusCompleted_Then_ReturnEmpty(
+                    [Frozen(Matching.ImplementedInterfaces)] ApprenticeFeedbackDataContext context
+                    )
+                {
+                    context.Add(AFT_Completed("Provider Name"));
+                    context.SaveChanges();
+
+                    var apprenticeFeedbackTargets = ((IApprenticeFeedbackTargetContext)context)
+                                    .Entities
+                                        .StatusNotCompletedOrPermanentlyWithdrawn()
+                                    .ToList();
+
+                    apprenticeFeedbackTargets.Should().BeEmpty();
+                }
+
+                [Test]
+                [AutoMoqData]
+                public void When_ApprenticeshipStatusWithdrawn_Then_ReturnEmpty(
+                    [Frozen(Matching.ImplementedInterfaces)] ApprenticeFeedbackDataContext context
+                    )
+                {
+                    context.Add(AFT_Withdrawn("Provider Name"));
+                    context.SaveChanges();
+
+                    var apprenticeFeedbackTargets = ((IApprenticeFeedbackTargetContext)context)
+                                    .Entities
+                                        .StatusNotCompletedOrPermanentlyWithdrawn()
+                                    .ToList();
+
+                    apprenticeFeedbackTargets.Should().BeEmpty();
+                }
+
+                [Test]
+                [AutoMoqData]
+                public void When_ApprenticeshipStatusNotCompletedOrWithdrawn_Then_ReturnApprenticeship(
+                    [Frozen(Matching.ImplementedInterfaces)] ApprenticeFeedbackDataContext context
+                    )
+                {
+                    context.Add(AFT_EligibilityCalculatedRecently("Provider Name"));
```

Comment:
Feels like the proving it works should always be the Valid Target, rather than others that just haven't got the right details specified. Just pondering that one 🤔